team/quiet.rst

```diff
@@ -0,0 +1,40 @@
+
+############
+Quiet Friday
+############
+
+Starting on 2020-09-25 DMLT will commence **Quiet Friday**. 
+
+Guidelines
+==========
+The driver for the quiet day is to allow staff a block of time without interruptions or the need to be online and receptive. 
+By reducing traffic we will reduce FOMO on discussion topics. 
+Hence there should be: 
+
+- No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasional we may be required to attend such meetings.
+- No Slack messages on general channels. You may slack an individual, or a group of individuals if you have agreed such interactions ahead of time. Conversations should not be carried out on general channels. There should be no expectation of response to Slack messages.
+
+Asynchronous communication is perfectly acceptable on the understanding than no response is to be expected before the next work day (Monday).
+Hence it is ok to, for example:
+
+- Send email.  Though you may consider using a delayed send.
+- Update Jira issues, which will result in email notifications that are ok. 
+- Perform github work. Code reviews, merges etc. which will result in notifications are ok.
+
+This will be generally publicized outside DM hence it is ok to set up an out office reply.
```

Comment:
Maybe be cautious here? We might tell other parts of Rubin about the quiet day, but they'll (probably) forget, and we can't expect the wider community to be aware of it.

(Arguably, the wider community can't expect us to be instantly responsive, of course.)